examples/profile/profile.py

```diff
@@ -0,0 +1,157 @@
+import argparse
+import os
+import shutil
+
+import chainer
+from chainer import cuda
+from chainer import function_hooks
+from chainer import links
+from chainer import optimizers
+import numpy
+import six
+
+import alex
+import conv
+import overfeat
+import vgg
+
+
+parser = argparse.ArgumentParser(description='Measurement of mean elapsed time of forward, '
+                                 'backward and parameter update.')
+parser.add_argument('--predictor', '-p', type=str, default='alex',
+                    choices=('alex', 'overfeat', 'vgg', 'conv1', 'conv2',
+                             'conv3', 'conv4', 'conv5'),
+                    help='network architecture')
+parser.add_argument('--seed', '-s', type=int, default=0,
+                    help='random seed')
+parser.add_argument('--iteration', '-i', type=int, default=10,
+                    help='the number of iterations to be averaged over.')
+parser.add_argument('--gpu', '-g', type=int, default=-1,
+                    help='GPU to use. Negative value to use CPU')
+parser.add_argument('--cudnn', '-c', action='store_true',
+                    help='True if using cudnn')
+parser.add_argument('--cache-level', '-C', type=str, default=None,
+                    choices=(None, 'memory', 'disk'),
+                    help='If `None`, memory cache and disk cache are removed '
```

Comment:
Can we pass None value explicitly? If not, you should write the help message without using this word, like "by default, ..." (since the default values are not printed in the help message).